contract_interface.md

```diff
@@ -38,3 +41,13 @@ The method exported as `main` will be executed by the VM.
 On successful execution, the code should return via a normal code path.
 
 If it needs to abort due to a failure, an *unreachable* instruction should be executed.
+
+### Start function
+
+The use of a [start function](https://webassembly.github.io/spec/core/syntax/modules.html#start-function) is disallowed.
+
+The reason for this is that an eWASM VM would need to have access to the memory space of a contract and that must be acquired prior to executing it.
```

Comment:
`ewasm`

Reply:
I kept it intentionally the old styling in this PR. There's the other PR changing all.

Reply:
ok I will update that PR when this is merged